src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/default.phpt

```diff
@@ -31,14 +31,14 @@ class FooTestCase
     public function testLegacyFoo()
     {
         @trigger_error('silenced foo deprecation', E_USER_DEPRECATED);
-        trigger_error('unsilenced foo deprecation', E_USER_DEPRECATED);
-        trigger_error('unsilenced foo deprecation', E_USER_DEPRECATED);
+        @trigger_error('unsilenced foo deprecation', E_USER_DEPRECATED);
+        @trigger_error('unsilenced foo deprecation', E_USER_DEPRECATED);
```

Comment:
I messed up with my second commit, but I've reverted that afterwards. Sorry for the noise.